### PR TITLE
Documentation Changes around Window Sizing and Clarification

### PR DESCRIPTION
**Pull request by anthony-ngu**
Changed the bootstrap.css as well as header and index pages to scale for window sizes correctly
Changed the microSD card steps for better clarification
Updated the remote debugging steps for better clarification, including images.
Changed the ConnectDownloadClarification.png in order to reflect the MSI download

### CODE REVIEW

File: SetupGalileo.md [hunk 1]
```diff
@@ -46,7 +46,8 @@ permalink: /SetupGalileo.htm
 
   <h2>Insert the microSD card</h2>
   <ol>
-    <li>Insert the microSD card with the Windows Image (provided in the box) into the Galileo's microSD card slot near the power port. (Note: This step must be done prior to plugging in the power.)</li>
+    <li>Remove the microSD card with the Windows Image form the SD card adapter.</li>
```
Comment (JustAddWires) [inline SetupGalileo.md]: "form the SD card adapter" should be "from the SD card adapter"

**Comment by JustAddWires** [comment]
This version still overwrites the header with menu items at medium width.  Not ready to pull.

File: SetupGalileo.md (hunk 1 above)
Comment (anthony-ngu) [inline SetupGalileo.md]: Woops! fixed in an updated commit

**Comment by anthony-ngu** [comment]
Since I was working out of a branch, my changes don't show up on blindnas.github.io/content

**Comment by JustAddWires** [comment]
This now looks ok.  Ok to pull.

**Pull request merged**